apps/expo/src/components/NavBar.tsx

```diff
@@ -73,6 +73,12 @@ const NavBar = () => {
           tabBarLabel: "Account",
         }}
       />
+      <Tabs.Screen
+        name="signin" // the signin screen should not be shown as a tab
+        options={{
+          href: null,
+        }}
+      />
```

Comment:
remove

Reply:
Why? This is needed to hide the tab at the bottom